docs/pages/docs/core-concepts/remote-caching.mdx

```diff
@@ -9,13 +9,23 @@ import { Tabs, Tab } from "../../../components/Tabs";
 
 # Remote Caching
 
-A major key to Turborepo's speed is that it is both lazy and efficient—it does the least amount of work possible and it tries to never redo work that's already been done before.
+Turborepo's [task cache](/docs/core-concepts/caching) can save a lot of time by never doing the same work twice.
 
-By default, Turborepo caches the outputs of your tasks on your machine's local file system. We call this "single-player mode." But what if you could take advantage of the computational work done by your teammates or your CI by sharing a single cache across machines? In other words, what if you could turn on "co-op multiplayer mode", or a "Dropbox" for your Turborepo cache?
+But there's an issue - the **cache is local to your machine**. When you're working with a CI, this can result in a lot of duplicated work:
```

Comment:
```suggestion
But there's an issue - the **cache is local to your machine**. This means that builds on your CI system can't use it.
```

Reply:
I prefer my own version, because it sets up the next image.